scala/lambdas/ingest-mapper/src/test/scala/uk/gov/nationalarchives/ingestmapper/DiscoveryServiceTest.scala

```diff
@@ -119,11 +119,11 @@ class DiscoveryServiceTest extends AnyFlatSpec {
 
     discoveryAsset.title should equal(None)
     discoveryAsset.citableReference should equal("A")
-    discoveryAsset.scopeContent.description should equal("")
+    discoveryAsset.scopeContent.description should equal(None)
```

Comment:
I'm unable to leave a comment on the test name but should the test name have "an empty title and description" in it?